src/lib/dnssd/tests/TestTxtFields.cpp

```diff
@@ -177,6 +191,28 @@ TEST(TestTxtFields, TestGetCommissioningMode)
     EXPECT_EQ(GetCommissioningMode(GetSpan(cm)), 0);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+TEST(TestTxtFields, TestGetJointFabricMode)
+{
+    char jfm[64];
+    strcpy(jfm, "0");
+    EXPECT_EQ(GetJointFabricMode(GetSpan(jfm)).Raw(), 0);
```

Comment:
would it make sense to not compare with RAW but rather bit flags to make it clear what we expect?

Ideally compare with 0 bits set, 1 bit set and then multiple ones.